Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -2506,6 +2507,7 @@ AODProducerWorkflowDPL::TrackQA AODProducerWorkflowDPL::processBarrelTrackQA(int
         byteMask |= (1 << i);
       }
     }
+    trackQAHolder.tpcTime0 = tpcOrig.getTime0();
```

Comment:
@miranov25 The TPC t0 is defined in timebins wrt stat of the timeframe, which is not stored in the AOD. Therefore, simply storing the getTime0 makes no sense as the reference is not defined. As we discussed before, since your QA track refers on the AOD track, the most convenient is to define t0 wrt track time stored in the AOD. 
The most straightforward would be to 
1) add to https://github.com/AliceO2Group/AliceO2/blob/c4a7b507c1288a50c75e04ebb2af49862a05fc0e/Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h#L387 
```
int diffBCRef = 0;  // offset of time reference BC from the start of the orbit
```
2) set it at https://github.com/AliceO2Group/AliceO2/blob/c4a7b507c1288a50c75e04ebb2af49862a05fc0e/Detectors/AOD/src/AODProducerWorkflowSpec.cxx#L2387 as
```
extraInfoHolder.diffBCRef = int(bcOfTimeRef - mStartIR.toLong());
```
3) pass the reference to extraInfoHolder to your `processBarrelTrackQA(...` method and 
define the time0 at https://github.com/AliceO2Group/AliceO2/blob/c4a7b507c1288a50c75e04ebb2af49862a05fc0e/Detectors/AOD/src/AODProducerWorkflowSpec.cxx#L2510 as
```
trackQAHolder.tpcTime0 = (tpcOrig.getTime0()*8 - extraInfoHolder.diffBCRef)*o2::constants::lhc::LHCBunchSpacingNS - extraInfoHolder.trackTime; // difference between TPC track time0 and stored track nominal time in ns
```

Reply:
@shahor02 
Thank you for your comment. I now have a clearer understanding of your previous comment on Mattermost. I will update it there according to your instructions after test. Hopefully, this will be done before the OFFLINE meeting.

Reply:
I incorporated your suggestion. I need to verify if the absolute precision is sufficient since this reference has a significant offset.

<img width="1128" alt="image" src="https://github.com/AliceO2Group/AliceO2/assets/21962765/99145aba-f6cd-4298-8fee-fa6ba58eb879">

Reply:
Time0 offset looks to be suspicious. I will run the debugger, looks like something not defined ....

Reply:
Fixed